hypershift-operator/controllers/scheduler/dedicated_request_serving_nodes.go

```diff
@@ -539,6 +633,49 @@ func (r *DedicatedServingComponentSchedulerAndSizer) Reconcile(ctx context.Conte
 	return ctrl.Result{}, nil
 }
 
+func (r *DedicatedServingComponentSchedulerAndSizer) ensurePairConfigMap(ctx context.Context, pairLabel, hcNamespace, hcName string) error {
+	cfgMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: placeholderNamespace,
+			Name:      pairLabel,
```

Comment:
Doesn't this leave open the issue of two reconcilers allocating the same cluster to two different pairs?

Reply:
> Doesn't this leave open the issue of two reconcilers allocating the same cluster to two different pairs?

Yes, but I would think the effect of that is not as bad as having 2 clusters with the same pair. We will eventually label/taint nodes in one of those pairs and that's what we will use.